packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -141,7 +141,9 @@ module.exports = (themeOptions = {}) => {
                 // It's important to specify the maxWidth (in pixels) of
                 // the content container as this plugin uses this as the
                 // base for generating different widths of each image.
-                maxWidth: 770,
+                maxWidth: 754,
+                loading: 'auto',
+                srcSetBreakpoints: [754, 1508],
```

Comment:
Let's add a comment/reference that these values are also used in the static query `use-resized-images.js`, in case we need to change the values.